cni/network/network.go

```diff
@@ -685,7 +685,10 @@ func (plugin *NetPlugin) findMasterInterface(opt *createEpInfoOpt) string {
 		return plugin.findMasterInterfaceBySubnet(opt.ipamAddConfig.nwCfg, &opt.ifInfo.HostSubnetPrefix)
 	case cns.NodeNetworkInterfaceFrontendNIC, cns.NodeNetworkInterfaceAccelnetFrontendNIC:
 		return plugin.findInterfaceByMAC(opt.ifInfo.MacAddress.String())
-	case cns.BackendNIC: // TODO: how to find interface with IB NIC by mac address
+	case cns.BackendNIC:
+		// if windows swiftv2 has right network drivers, there will be an NDIS interface while the VFs are mounted
+		// when the VF is dismounted, this interface will go away
+		// return an unique interface name to containerd
 		opt.ifInfo.Name = ibInterfacePrefix + strconv.Itoa(opt.endpointIndex)
```

Comment:
I think I missed this change in another PR but I don't think we should be modifying if info in this function. It's only meant to _find_ the master interface name-- there shouldn't be any side effects (like modifying it opt.ifInfo.Name) or we might be left wondering how finding the master interface has modified some value.

Reply:
Doesn't this: `opt.ifInfo.Name = ibInterfacePrefix + strconv.Itoa(opt.endpointIndex)` modify opt.ifInfo.Name? If you did not intend to modify `opt.ifInfo.Name` maybe in a future PR we could just return `ibInterfacePrefix + strconv.Itoa(opt.endpointIndex)` directly without storing it in an intermediate variable.